charts/rancher-k3s-upgrader/0.3.1/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
This is just a chart that was merged to dev-v2.6 since we updated release branch ?

Reply:
Yes, that's correct.